src/reducer/database/helper/statistic.js

```diff
@@ -0,0 +1,49 @@
+import moment from "moment";
+
+function calculateStreaks(values) {
+  if (!values) {
```

Comment:
What is the use case of this, can't you just return an empty list if the values are empty?